pinecone/index_connection.go

```diff
@@ -996,8 +1003,307 @@ func (idx *IndexConnection) DescribeIndexStatsFiltered(ctx context.Context, meta
 	}, nil
 }
 
-func (idx *IndexConnection) query(ctx context.Context, req *data.QueryRequest) (*QueryVectorsResponse, error) {
-	res, err := (*idx.dataClient).Query(idx.akCtx(ctx), req)
+// StartImportResponse holds the response parameters for the StartImport method.
+//
+// Fields:
+//   - Id: The ID of the import process that was started.
+type StartImportResponse struct {
+	Id string `json:"id,omitempty"`
+}
+
+// StartImport imports data from a storage provider into an index. The uri parameter must start with the
+// schema of a supported storage provider. For buckets that are not publicly readable, you will also need to
+// separately configure a storage integration and pass the integration id.
```

Comment:
Do we have instructions in our docs for how to build a storage integration? If we're telling ppl they need to do an extra step, it might be nice for us to provide them resources about how to do that in the same sentence

Reply:
Added a link to: https://docs.pinecone.io/guides/operations/integrations/manage-storage-integrations